### DISCUSSION TITLE
Tuxedo OS Logo

### DISCUSSION THREAD

**Discussion by RoiBurgonde** [fastfetch-cli/fastfetch#528] [Q&A] [answered]
Hi,
Do you think it is possible to add Tuxedo OS logo ?

Thanx in advance for your answer.

**Comment by CarterLi** [comment]
Yeap, if you provide the logo source.

**Comment by RoiBurgonde** [comment]
Ok. I don't know how where to find the logo source. But it exists in neofetch. Maybe it is possible to take the logo from it?
If you explain me how to, maybe i could provide you the logo ?

**Reply by CarterLi** [reply]
> But it exists in neofetch

Where's it?

**Comment by RoiBurgonde** [comment]
Wow, it's weird, it appears in the list (neofetch --help) on my laptop which uses Tuxedo OS, but not on my other PC which uses Linux Mint... :S

**Reply by CarterLi** [reply]
I think it is customized by Tuxedo developers.

You may paste the content of `command -v neofetch` here

**Comment by RoiBurgonde** [comment]
Well, it just returns /usr/bin/neofetch

**Reply by CarterLi** [reply]
Yes, upload it here

**Reply by RoiBurgonde** [reply]
Here it is:

[neofetch.tar.gz](https://github.com/fastfetch-cli/fastfetch/files/12397091/neofetch.tar.gz)

**Comment by CarterLi** [comment]
Done. You may test it with `fastfetch -l 'Tuxedo OS'`

**Reply by RoiBurgonde** [reply]
I installed fastfetch with the deb file. I suppose i need to download the new version ?

**Reply by CarterLi** [reply]
Yes

**Reply by RoiBurgonde** [reply]
Well, 2.0.1 deb file appears to be upload 12 hours ago. Do i need to wait ?

**Reply by CarterLi** [reply, marked as answer]
Download the dev build here

https://github.com/fastfetch-cli/fastfetch/actions/runs/5927836875

**Reply by RoiBurgonde** [reply]
Well, it works perfectly!! Thank you very much for the time spent !!
